jenkins/image_building/upload-ci-image.sh

```diff
@@ -26,17 +26,10 @@ upload_ci_image() {
 
   img_name="$1"
 
-  # Push image to openstack Kna region
-  openstack image create "${img_name}" --file "${img_name}".qcow2 --disk-format=qcow2
+  # Push image to openstack xerces
+  export OS_AUTH_URL="https://xerces.ericsson.net:5000"
+  openstack image create "${img_name}" --file "${img_name}".raw --disk-format=raw
```

Comment:
Here the filename magically does have `.raw` suffix. Original Qcow2 implementation has the suffix in both places?

Reply:
Yes but here we dont need in both places